tools/image-builder/files_stargz/etc/containerd-stargz-grpc/config.toml

```diff
@@ -0,0 +1,2 @@
+metrics_address = "localhost:8080"
```

Comment:
Can we choose a port other than 8080? I'd like to reduce the chance of future conflicts. Stargz's examples use 8234